.github/workflows/aws-upload-layer-on-release.yml

```diff
@@ -67,8 +67,8 @@ jobs:
           release_layer: 'kensu-py--dev'
         run: |
           mkdir python
-          pip install -r requirements.txt -t python
-          pip install . -t  python
+          pip install -e ".[no-extra-deps]"
```

Comment:
FYI: for AWS layer we need different deps - old `requirements.txt` excluded the `pandas`, `numpy` as these to prevent runtime conflicts/issues need to be added as binary AWS layers separately.

Also it didn't include any fancy optional libs, so most likely we need only basic dependencies and NOTHING from `extra-requirements.txt`, so I guess a new "option" is needed here which I tried to add it myself

Reply:
ok good to know, so this is working right? We can merge?

Reply:
yes, soon, just i'll need to sign/rebase your unsigned commits ;) and merge commits seem to create some mess.

Reply:
darn -_-
sorry